src/antares/service/local_services/link_local.py

```diff
@@ -133,3 +137,56 @@ def sort_link_properties_dict(ini_dict: Dict[str, str]) -> Dict[str, str]:
             "filter-year-by-year",
         ]
         return dict(sorted(ini_dict.items(), key=lambda item: dict_order.index(item[0])))
+
+    def get_capacity_direct(
+        self,
+        area_id: str,
```

Comment:
Also it should be typed area_from, area_to to fit with the existing naming we chose

Reply:
Fixed in the last commit